src/contexts/Plugins/index.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { PluginsList } from 'consts';
+import React, { useRef, useState } from 'react';
+import { localStorageOrDefault, setStateWithRef } from 'Utils';
+import * as defaults from './defaults';
+import { PluginsContextInterface } from './types';
+
+export const PluginsContext = React.createContext<PluginsContextInterface>(
+  defaults.defaultPluginsContext
+);
+
+export const usePlugins = () => React.useContext(PluginsContext);
+
+export const PluginsProvider = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  // get initial plugins
+  const getAvailablePlugins = () => {
+    // get plugins config from local storage
+    const _plugins: any = localStorageOrDefault('plugins', PluginsList, true);
```

Comment:
rename `_plugins` to `localPlugins`. We are moving away from using the underscore.